x-pack/plugin/eql/qa/common/src/main/resources/test_queries_date_nanos.toml

```diff
@@ -4,20 +4,20 @@
 [[queries]]
 name = "filterOnDateNanosWithMillis"
 query = '''
-    process where @timestamp == "1970-01-02T12:31:25.510Z"
+    process where @timestamp == "1974-03-02T19:53:16.510Z"
```

Comment:
Why this change in test data? I got from you test changes that the data itself didn't actually change, I'm curious why have you changed these tests and, at the same time, `expected_event_ids` remained the same.

Reply:
> I got from you test changes that the data itself didn't actually change

Yes, it did change: the nano tests used before a copy of `endgame-140.data` that was modified to match the few tests contained in `test_queries_date_nanos.toml`. With this PR `eql_date_nanos.data` has been removed and tests setup to use with the same data - `endgame-140.data` - [with different interpretation of the timestamp value](https://github.com/elastic/elasticsearch/pull/76953#discussion_r696092631). This allows to run the same tests - `test_queries.toml` on both millis and nanos. But this also requires the event nano tests - `test_queries_date_nanos.toml`  - changed in order to match the existing results (that are kept unchanged).